docs/source/60_building_apps.md

```diff
@@ -184,16 +200,21 @@ object MyExample {
       System.err.println("at least one argument required, e.g. input.foo")
       System.exit(1)
     }
+  }
+}
 ```
 
-Create an Apache Spark configuration `SparkConf` and use it to create a new `SparkContext`.
-The following serialization configuration needs to be present to register ADAM classes. If
-any additional [Kyro serializers](https://github.com/EsotericSoftware/kryo) need to be
-registered, [create a registrator that delegates to the ADAM registrator](#registrator).
-You might want to provide your own serializer registrator if you need custom serializers for
-a class in your code that either has a complex structure that Kryo fails to serialize properly
-via Kryo's serializer inference, or if you want to require registration of all classes in your
-application to improve performance.
+Create an Apache Spark configuration `SparkConf` and use it to create a new
+`SparkContext`. The following serialization configuration needs to be present
+to register ADAM classes. If any additional
+[Kyro serializers](https://github.com/EsotericSoftware/kryo) need to be
+registered,
+[create a registrator that delegates to the ADAM registrator](#registrator).
```

Comment:
From a word wrap perspective, you shouldn't need the whole link text to be on a single line, no?